docs/rst/use_cases/ros_cloud.rst

```diff
@@ -0,0 +1,148 @@
+that will play the client role in the discovery process of remote participants
+(see :term:`Discovery Server discovery mechanism <Discovery Server>`).
+
+After having acknowledged each other’s existence through `Simple DDS discovery mechanism <https://fast-dds.docs.eprosima.com/en/latest/fastdds/discovery/simple.html>`_
+(multicast communication), the local participant will start receiving messages published by the ROS 2 talker node, and
+will then forward them to the WAN participant. Following, these messages will be sent to another participant hosted on a
+|k8s| cluster to which it connects via WAN communication over UDP/IP.
+
+Following is a representation of the above-described scenario:
+
+.. figure:: /rst/figures/vulcanexus_local.png
+
+
+Local router
+------------
+The configuration file used by the local router will be the following:
+
+.. literalinclude:: ../../resources/use_cases/ros_cloud/local-ddsrouter.yaml
+    :language: yaml
+
+Note that the simple participant will be receiving messages sent in DDS domain ``0``. Also note that, due to the choice
+of UDP as transport protocol, a listening address with the LAN public IP address needs to be specified for the local WAN
+participant, even when behaving as client in the participant discovery process. Make sure that the given port is
+reachable from outside this local network by properly configuring port forwarding in your Internet router device.
+The connection address points to the remote WAN participant deployed in the |k8s| cluster. For further details on how to
+configure WAN communication, please have a look at :ref:`WAN Configuration <user_manual_wan_configuration>`.
+
+To launch the local router, execute:
+
+.. code-block:: bash
+
+    ddsrouter --config-path local-ddsrouter.yaml
+
+
+Talker
+------
+This example will make use of *ROS 2 galactic* with ``demo-nodes-cpp`` package installed. If not already present in your
+system, you may choose any of the available options to `install ROS galactic <https://docs.ros.org/en/galactic/Installation.html>`_,
+or even consider directly using a distributed `Docker image <https://hub.docker.com/_/ros>`_. Just make sure the
+resulting environment is prepared to utilize |efastdds| as middleware (see `Working with eProsima Fast DDS <https://docs.ros.org/en/galactic/Installation/DDS-Implementations/Working-with-eProsima-Fast-DDS.html>`_).
+
+Once *ROS 2* is installed, start publishing messages in DDS domain ``0`` by executing:
+
+.. code-block:: bash
+
+    RMW_IMPLEMENTATION=rmw_fastrtps_cpp ros2 run demo_nodes_cpp talker
+
+
+
+Kubernetes setup
+================
+Two different deployments will be used for this example, each in a different |k8s| pod. The |ddsrouter| cloud instance
+(cloud router) consists of two participants:
+
+* A WAN participant that receives the messages coming from our LAN through the aforementioned UDP communication channel.
```

Comment:
Add a link to WAN Participant (even if it already exist before)
If not, it does not look fine with the link below.

Reply:
:+1: